lib/jsdom/living/nodes/RadioNodeList-impl.js

```diff
@@ -0,0 +1,28 @@
+"use strict";
+
+const NodeListImpl = require("./NodeList-impl").implementation;
+
+class RadioNodeListImpl extends NodeListImpl {
+  get value() {
+    this._update();
+    const element = this._list.find(e => e._localName === "input" && e.type === "radio" && e.checked);
+    if (element && element.value !== "") {
+      return element.value ? element.value : "on";
+    }
+    return "";
+  }
+  set value(value) {
+    this._update();
```

Comment:
Again, this doesn't follow the spec very well. Please try to translate it more precisely.